---
subpage: true
title: "OsNHX1"
description: ""
category: genes
tags: [abiotic stress, salt, lateral root, shoot, salt tolerance, root, pollen, vascular bundle, salt stress, grain, seedling, tolerance, drought, grain yield, yield, drought tolerance, stomatal]
---

* **Information**  
    + Symbol: OsNHX1  
    + MSU: [LOC_Os07g47100](http://rice.plantbiology.msu.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os07g47100)  
    + RAPdb: [Os07g0666900](http://rapdb.dna.affrc.go.jp/viewer/gbrowse_details/irgsp1?name=Os07g0666900)  

* **Publication**  
    + [Molecular cloning and expression of the Na+/H+ exchanger gene in Oryza sativa](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular+cloning+and+expression+of+the+Na+/H++exchanger+gene+in+Oryza+sativa%5BTitle%5D), 1999, Biochim Biophys Acta.
    + [Salt-dependent regulation of chloride channel transcripts in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Salt-dependent+regulation+of+chloride+channel+transcripts+in+rice%5BTitle%5D), 2006, Plant Science.
    + [Molecular and functional analyses of rice NHX-type Na+/H+ antiporter genes](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular+and+functional+analyses+of+rice+NHX-type+Na+/H++antiporter+genes%5BTitle%5D), 2011, Planta.
    + [Function, intracellular localization and the importance in salt tolerance of a vacuolar Na+/H+ antiporter from rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Function,+intracellular+localization+and+the+importance+in+salt+tolerance+of+a+vacuolar+Na+/H++antiporter+from+rice%5BTitle%5D), 2004, Plant Cell Physiol.
    + [OsbZIP71, a bZIP transcription factor, confers salinity and drought tolerance in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=OsbZIP71,+a+bZIP+transcription+factor,+confers+salinity+and+drought+tolerance+in+rice%5BTitle%5D), 2014, Plant Mol Biol.
    + [Enhanced Salt Tolerance Conferred by the Complete 2.3 kb cDNA of the Rice Vacuolar Na+/H+ Antiporter Gene Compared to 1.9 kb Coding Region with 5' UTR in Transgenic Lines of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Enhanced+Salt+Tolerance+Conferred+by+the+Complete+2.3+kb+cDNA+of+the+Rice+Vacuolar+Na+/H++Antiporter+Gene+Compared+to+1.9+kb+Coding+Region+with+5'+UTR+in+Transgenic+Lines+of+Rice.%5BTitle%5D), 2016, Front Plant Sci.
    + [Alterations in stomatal response to fluctuating light increase biomass and yield of rice under drought conditions](http://www.ncbi.nlm.nih.gov/pubmed?term=Alterations+in+stomatal+response+to+fluctuating+light+increase+biomass+and+yield+of+rice+under+drought+conditions%5BTitle%5D), 2020, Plant Journal.
    + [Histone deacetylase HDA710 controls salt tolerance by regulating ABA signaling in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Histone+deacetylase+HDA710+controls+salt+tolerance+by+regulating+ABA+signaling+in+rice%5BTitle%5D), 2020, J Integr Plant Biol.
    + [Gene Expression Dynamics in Rice Peduncles at the Heading Stage](http://www.ncbi.nlm.nih.gov/pubmed?term=Gene+Expression+Dynamics+in+Rice+Peduncles+at+the+Heading+Stage%5BTitle%5D), 2020, Front Genet.
    + [Genome-Wide Association Study GWAS to Identify Salt-Tolerance QTLs Carrying Novel Candidate Genes in Rice During Early Vegetative Stage](N+Y).

* **Genbank accession number**  
    + [AB021878](http://www.ncbi.nlm.nih.gov/nuccore/AB021878)

* **Key message**  
    + Real-time PCR analysis revealed that the abiotic stress-related genes, OsVHA-B, OsNHX1, COR413-TM1, and OsMyb4, were up-regulated in overexpressing lines, while these same genes were down-regulated in RNAi lines
    + The expression of OsNHX1, OsNHX2, OsNHX3, and OsNHX5 is regulated differently in rice tissues and is increased by salt stress, hyperosmotic stress, and ABA
    + When we studied the expression of beta-glucuronidase (GUS) driven by either the OsNHX1 or the OsNHX5 promoter, we observed activity in the stele, the emerging part of lateral roots, the vascular bundle, the water pore, and the basal part of seedling shoots with both promoters
    + These results suggest that the product of the novel gene, OsNHX1, functions as a Na+/H+ exchanger, and plays important roles in salt tolerance of rice
    + In addition, overexpression of OsNHX1 improved the salt tolerance of transgenic rice cells and plants
    + These results suggest that OsNHX1 on the tonoplasts plays important roles in the compartmentation of Na(+) and K(+) highly accumulated in the cytoplasm into the vacuoles, and the amount of the antiporter is one of the most important factors determining salt tolerance in rice
    + OsNHX1 promoter-GUS activity only was localized to the guard cells and trichome, whereas OsNHX5 promoter-GUS activity only was localized to the root tip and pollen grains
    + Treatment with high concentrations of NaCl and KCl increased the transcript levels of OsNHX1 in rice roots and shoots
    + It also shows that inclusion of the complete un-translated regions (UTRs) of the alternatively spliced OsNHX1 gene provides a higher level of tolerance to the transgenic rice
    + OsNHX1 overexpression in rice resulted in a decrease in ¦Ócl , and an increase in biomass, grain yield under drought
    + Taken together, the natural variation in OsNHX1 could be utilized to manipulate the stomatal dynamics for an improved rice drought tolerance
    + Herein, using genome-wide association study, we showed that the genetic variation in OsNHX1 was strongly associated with the changes in ¦Ócl , the time constant of stomatal closure, in 206 rice accessions

* **Connection**  
    + [Molecular and functional analyses of rice NHX-type Na+/H+ antiporter genes](OsNHX1+through+OsNHX4+and+OsNHX5))
    + [Molecular and functional analyses of rice NHX-type Na+/H+ antiporter genes](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular+and+functional+analyses+of+rice+NHX-type+Na+/H++antiporter+genes%5BTitle%5D), OsNHX1, OsNHX2, OsNHX3, and OsNHX5 can suppress the Na+, Li+, and hygromycin sensitivity of yeast nhx1 mutants and their sensitivity to a high K+ concentration
    + [Molecular and functional analyses of rice NHX-type Na+/H+ antiporter genes](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular+and+functional+analyses+of+rice+NHX-type+Na+/H++antiporter+genes%5BTitle%5D), The expression of OsNHX1, OsNHX2, OsNHX3, and OsNHX5 is regulated differently in rice tissues and is increased by salt stress, hyperosmotic stress, and ABA
    + [Molecular and functional analyses of rice NHX-type Na+/H+ antiporter genes](GUS) driven by either the OsNHX1 or the OsNHX5 promoter, we observed activity in the stele, the emerging part of lateral roots, the vascular bundle, the water pore, and the basal part of seedling shoots with both promoters
    + [Molecular and functional analyses of rice NHX-type Na+/H+ antiporter genes](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular+and+functional+analyses+of+rice+NHX-type+Na+/H++antiporter+genes%5BTitle%5D), OsNHX1 promoter-GUS activity only was localized to the guard cells and trichome, whereas OsNHX5 promoter-GUS activity only was localized to the root tip and pollen grains
    + [Molecular and functional analyses of rice NHX-type Na+/H+ antiporter genes](OsNHX1+through+OsNHX4+and+OsNHX5))
    + [Molecular and functional analyses of rice NHX-type Na+/H+ antiporter genes](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular+and+functional+analyses+of+rice+NHX-type+Na+/H++antiporter+genes%5BTitle%5D), OsNHX1, OsNHX2, OsNHX3, and OsNHX5 can suppress the Na+, Li+, and hygromycin sensitivity of yeast nhx1 mutants and their sensitivity to a high K+ concentration
    + [Molecular and functional analyses of rice NHX-type Na+/H+ antiporter genes](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular+and+functional+analyses+of+rice+NHX-type+Na+/H++antiporter+genes%5BTitle%5D), The expression of OsNHX1, OsNHX2, OsNHX3, and OsNHX5 is regulated differently in rice tissues and is increased by salt stress, hyperosmotic stress, and ABA
    + [OsbZIP71, a bZIP transcription factor, confers salinity and drought tolerance in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=OsbZIP71,+a+bZIP+transcription+factor,+confers+salinity+and+drought+tolerance+in+rice%5BTitle%5D), Real-time PCR analysis revealed that the abiotic stress-related genes, OsVHA-B, OsNHX1, COR413-TM1, and OsMyb4, were up-regulated in overexpressing lines, while these same genes were down-regulated in RNAi lines
    + [OsbZIP71, a bZIP transcription factor, confers salinity and drought tolerance in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=OsbZIP71,+a+bZIP+transcription+factor,+confers+salinity+and+drought+tolerance+in+rice%5BTitle%5D), Chromatin immunoprecipitation analysis confirmed that OsbZIP71 directly binds the promoters of OsNHX1 and COR413-TM1 in vivo
    + [OsPEX11, a Peroxisomal Biogenesis Factor 11, Contributes to Salt Stress Tolerance in Oryza sativa.](+) homeostasis in transgenic plants under salinity
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](OsNIN-like2,+OsNIN-like3+and+OsNIN-like4) binding to the OsNHX1 gene promoter
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](OsNIN-like2,+OsNIN-like3+and+OsNIN-like4) were identified as binding to OsNHX1 promoter
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](http://www.ncbi.nlm.nih.gov/pubmed?term=Five+novel+transcription+factors+as+potential+regulators+of+OsNHX1+gene+expression+in+a+salt+tolerant+rice+genotype.%5BTitle%5D), Transactivation activity assays performed in Arabidopsis and rice protoplasts showed that OsPCF2 and OsNIN-like4 are activators of the OsNHX1 gene expression, while OsCPP5 and OsNIN-like2 act as repressors
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](OsNIN-like2,+OsNIN-like3+and+OsNIN-like4) binding to the OsNHX1 gene promoter
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](OsNIN-like2,+OsNIN-like3+and+OsNIN-like4) were identified as binding to OsNHX1 promoter
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](http://www.ncbi.nlm.nih.gov/pubmed?term=Five+novel+transcription+factors+as+potential+regulators+of+OsNHX1+gene+expression+in+a+salt+tolerant+rice+genotype.%5BTitle%5D), Transactivation activity assays performed in Arabidopsis and rice protoplasts showed that OsPCF2 and OsNIN-like4 are activators of the OsNHX1 gene expression, while OsCPP5 and OsNIN-like2 act as repressors
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](OsNIN-like2,+OsNIN-like3+and+OsNIN-like4) binding to the OsNHX1 gene promoter
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](OsNIN-like2,+OsNIN-like3+and+OsNIN-like4) were identified as binding to OsNHX1 promoter
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](OsNIN-like2,+OsNIN-like3+and+OsNIN-like4) binding to the OsNHX1 gene promoter
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](OsNIN-like2,+OsNIN-like3+and+OsNIN-like4) were identified as binding to OsNHX1 promoter
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](http://www.ncbi.nlm.nih.gov/pubmed?term=Five+novel+transcription+factors+as+potential+regulators+of+OsNHX1+gene+expression+in+a+salt+tolerant+rice+genotype.%5BTitle%5D), Transactivation activity assays performed in Arabidopsis and rice protoplasts showed that OsPCF2 and OsNIN-like4 are activators of the OsNHX1 gene expression, while OsCPP5 and OsNIN-like2 act as repressors
    + [Five novel transcription factors as potential regulators of OsNHX1 gene expression in a salt tolerant rice genotype.](http://www.ncbi.nlm.nih.gov/pubmed?term=Five+novel+transcription+factors+as+potential+regulators+of+OsNHX1+gene+expression+in+a+salt+tolerant+rice+genotype.%5BTitle%5D), In addition, although the OsNHX1 response to salt and PEG-induced drought stress in either shoots or roots was quite similar in both rice genotypes, the expression of OsPCF2 in roots under salt stress and the OsNIN-like4 in roots subjected to PEG was mainly up-regulated in Hasawi, indicating that these TFs may be associated with the salt and drought stress tolerance observed for this genotype
    + [Expression levels of the Na+/K+ transporter OsHKT2;1 and vacuolar Na+/H+ exchanger OsNHX1, Na enrichment, maintaining the photosynthetic abilities and growth performances of indica rice seedlings under salt stress.](http://www.ncbi.nlm.nih.gov/pubmed?term=Expression+levels+of+the+Na+/K++transporter+OsHKT2;1+and+vacuolar+Na+/H++exchanger+OsNHX1,+Na+enrichment,+maintaining+the+photosynthetic+abilities+and+growth+performances+of+indica+rice+seedlings+under+salt+stress.%5BTitle%5D), Expression levels of the Na+/K+ transporter OsHKT2;1 and vacuolar Na+/H+ exchanger OsNHX1, Na enrichment, maintaining the photosynthetic abilities and growth performances of indica rice seedlings under salt stress.
    + [Expression levels of the Na+/K+ transporter OsHKT2;1 and vacuolar Na+/H+ exchanger OsNHX1, Na enrichment, maintaining the photosynthetic abilities and growth performances of indica rice seedlings under salt stress.](200<U+00A0>mM+NaCl) in two rice cultivars, i
    + [The trihelix transcription factor OsGT¦Ã-2 is involved adaption to salt stress in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=The+trihelix+transcription+factor+OsGT¦Ã-2+is+involved+adaption+to+salt+stress+in+rice.%5BTitle%5D),  Several major genes responsible for ion transporting, such as the OsHKT2; 1, OsHKT1; 3 and OsNHX1 were transcriptionally regulated by OsGT¦Ã-2
    + [The trihelix transcription factor OsGT¦Ã-2 is involved adaption to salt stress in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=The+trihelix+transcription+factor+OsGT¦Ã-2+is+involved+adaption+to+salt+stress+in+rice.%5BTitle%5D),  A subsequent yeast one-hybrid assay and EMSA indicated that OsGT¦Ã-2 directly interacted with the promoters of OsHKT2; 1, OsNHX1 and OsHKT1; 3
    + [Histone deacetylase HDA710 controls salt tolerance by regulating ABA signaling in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Histone+deacetylase+HDA710+controls+salt+tolerance+by+regulating+ABA+signaling+in+rice%5BTitle%5D),  Moreover, ABA- and salt-stress-responsive genes, such as OsLEA3, OsABI5, OsbZIP72, and OsNHX1, were upregulated in hda710 compared with wild-type plants
    + [Histone deacetylase HDA710 controls salt tolerance by regulating ABA signaling in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Histone+deacetylase+HDA710+controls+salt+tolerance+by+regulating+ABA+signaling+in+rice%5BTitle%5D),  Moreover, ABA- and salt-stress-responsive genes, such as OsLEA3, OsABI5, OsbZIP72, and OsNHX1, were upregulated in hda710 compared with wild-type plants
    + [Histone deacetylase HDA710 controls salt tolerance by regulating ABA signaling in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Histone+deacetylase+HDA710+controls+salt+tolerance+by+regulating+ABA+signaling+in+rice%5BTitle%5D),  Moreover, ABA- and salt-stress-responsive genes, such as OsLEA3, OsABI5, OsbZIP72, and OsNHX1, were upregulated in hda710 compared with wild-type plants
    + [Histone deacetylase HDA710 controls salt tolerance by regulating ABA signaling in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Histone+deacetylase+HDA710+controls+salt+tolerance+by+regulating+ABA+signaling+in+rice%5BTitle%5D),  Moreover, ABA- and salt-stress-responsive genes, such as OsLEA3, OsABI5, OsbZIP72, and OsNHX1, were upregulated in hda710 compared with wild-type plants



