---
subpage: true
title: "OsDXS,OsDXS3"
description: ""
category: genes
tags: [pathogen]
---

* **Information**  
    + Symbol: OsDXS,OsDXS3  
    + MSU: [LOC_Os07g09190](http://rice.plantbiology.msu.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os07g09190)  
    + RAPdb: [Os07g0190000](http://rapdb.dna.affrc.go.jp/viewer/gbrowse_details/irgsp1?name=Os07g0190000)  

* **Publication**  
    + [Elicitor induced activation of the methylerythritol phosphate pathway toward phytoalexins biosynthesis in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Elicitor+induced+activation+of+the+methylerythritol+phosphate+pathway+toward+phytoalexins+biosynthesis+in+rice%5BTitle%5D), 2007, Plant Mol Biol.
    + [Identification of a small set of genes commonly regulated in rice roots in response to beneficial rhizobacteria](http://www.ncbi.nlm.nih.gov/pubmed?term=Identification+of+a+small+set+of+genes+commonly+regulated+in+rice+roots+in+response+to+beneficial+rhizobacteria%5BTitle%5D), 2020, Physiol Mol Biol Plants.

* **Genbank accession number**  
    + [AK100909](http://www.ncbi.nlm.nih.gov/nuccore/AK100909)

* **Key message**  
    + Interestingly, exposure to a rice bacterial pathogen also triggered the expression of OsDXS3 while the expression of Os02g0582900 and PR1b was down-regulated, suggesting that these genes might play a key role in rice-bacteria interactions

* **Connection**  
    + [Elicitor induced activation of the methylerythritol phosphate pathway toward phytoalexins biosynthesis in rice](CMS), which catalyze the first three committed steps in the MEP pathway, were further shown to have enzymatic activities that complement the growth of E
    + [Elicitor induced activation of the methylerythritol phosphate pathway toward phytoalexins biosynthesis in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Elicitor+induced+activation+of+the+methylerythritol+phosphate+pathway+toward+phytoalexins+biosynthesis+in+rice%5BTitle%5D), Application of ketoclomazone and fosmidomycin, inhibitors of DXS and DXR, respectively, repressed the accumulation of diterpene-type phytoalexins in suspension cells treated with chitin elicitor
    + [Identification of target genes of the bZIP transcription factor OsTGAP1, whose overexpression causes elicitor-induced hyperaccumulation of diterpenoid phytoalexins in rice cells.](http://www.ncbi.nlm.nih.gov/pubmed?term=Identification+of+target+genes+of+the+bZIP+transcription+factor+OsTGAP1,+whose+overexpression+causes+elicitor-induced+hyperaccumulation+of+diterpenoid+phytoalexins+in+rice+cells.%5BTitle%5D), A subsequent transactivation assay further confirmed the direct regulation of OsDXS3 expression by OsTGAP1, but other MEP pathway genes were not included among the OsTGAP1 target genes
    + [OsTGAP1 is responsible for JA-inducible diterpenoid phytoalexin biosynthesis in rice roots with biological impacts on allelopathic interaction.](http://www.ncbi.nlm.nih.gov/pubmed?term=OsTGAP1+is+responsible+for+JA-inducible+diterpenoid+phytoalexin+biosynthesis+in+rice+roots+with+biological+impacts+on+allelopathic+interaction.%5BTitle%5D),  Reporter analysis in planta revealed that OsTGAP1 activated the promoters of OsDXS3 and momilactone biosynthetic gene OsKSL4, presumably through binding to the TGACGT motif
    + [Identification of a small set of genes commonly regulated in rice roots in response to beneficial rhizobacteria](PR1b,+Os01g0382000) was significantly repressed
    + [Identification of a small set of genes commonly regulated in rice roots in response to beneficial rhizobacteria](PR1b,+Os01g0382000) was significantly repressed
    + [Identification of a small set of genes commonly regulated in rice roots in response to beneficial rhizobacteria](http://www.ncbi.nlm.nih.gov/pubmed?term=Identification+of+a+small+set+of+genes+commonly+regulated+in+rice+roots+in+response+to+beneficial+rhizobacteria%5BTitle%5D),  Interestingly, exposure to a rice bacterial pathogen also triggered the expression of OsDXS3 while the expression of Os02g0582900 and PR1b was down-regulated, suggesting that these genes might play a key role in rice-bacteria interactions



